ui/components/component-library/modal-body/modal-body.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  BackgroundColor,
+  Display,
+  FlexDirection,
+} from '../../../helpers/constants/design-system';
+import { Text } from '..';
+import README from './README.mdx';
+import { ModalBody } from './modal-body';
+
+const meta: Meta<typeof ModalBody> = {
+  title: 'Components/ComponentLibrary/ModalBody',
+  component: ModalBody,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    className: {
+      control: 'text',
+    },
+    children: {
+      control: 'text',
+    },
+  },
+  args: {
+    className: '',
+    children: 'Modal Body',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ModalBody>;
+
+export const DefaultStory: Story = {};
+
+DefaultStory.storyName = 'Default';
+
+export const Children: Story = {
+  args: {
+    className: '',
```

Comment:
Remove this
```suggestion
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/22154/commits/8e21ac564895f55f2eb69bfa7644db9d787e266c